example/workflow/unit/io/download_from_object_storage.json

```diff
@@ -0,0 +1,14 @@
+{
+    "filename": "index.html",
+    "overwrite": true,
+    "objectData": {
+        "CONTAINER": "example-cluster-001",
+        "NAME": "/path/to/the/index.html",
+        "PROVIDER": "some-cloud-compute-provider",
+        "REGION": "us-west-1"
+    },
+    "flowchartId": "download-files",
+    "head": true,
+    "name": "download_files",
+    "type": "download_from_urls"
```

Comment:
is this still accurate?  Or is it something like `download_from_object_storage`?

Reply:
Type is io

Reply:
Fixed type to now be io